R/visualisation_recipe_internal.R

```diff
@@ -237,6 +244,27 @@
     l <- l + 1
   }
 
+  # only points, no error bars - when show_ci = FALSE
```

Comment:
I don't think you need a new separate layer when show_ci is FALSE, you can just replace the `pointrange` one's "geom" by `"point"`

Also I think currently if people modify  `point` list it modifies both the raw data points and these ones so it might be best to keep it under `pointrange`

Reply:
Does it work to keep it under point range? I thought it might warn or error because of the ymin/ymax aes, which points don't have?

Reply:
> Also I think currently if people modify `point` list it modifies both the raw data points and these ones so it might be best to keep it under `pointrange`

We could just skip modifying estimate points, but anyway, who wants to show row data and point geoms for predictions *without* error bars? ;-)
Hiding CIs is mostly intended for many line geoms, honestly.

Reply:
> error because of the ymin/ymax aes, which points don't have?

no it doesn't 

```r
x <- modelbased::estimate_means(lm(Sepal.Length ~ Species, data = iris))
plot(x, pointrange = list(geom="point"))
```
![image](https://github.com/user-attachments/assets/b3947a18-40fb-42d0-959b-c4076f9377e2)

Another option then is to allow for `ribbon = "none"` ?

Reply:
That sounds good!  Let me try later

Reply:
ok, I removed `show_ci` and just added a check for `ribbon = "none"` in #361